doc/classes/Control.xml

```diff
@@ -85,6 +85,13 @@
 				[/codeblocks]
 			</description>
 		</method>
+		<method name="_get_accessibility_container_name" qualifiers="virtual const">
+			<return type="String" />
+			<param index="0" name="node" type="Node" />
+			<description>
+				Return a human-readable description of the position of [param node] child in the custom container, added to the node name.
```

Comment:
```suggestion
				Override this method to return a human-readable description of the position of the child [param node] in the custom container, added to the node name.
```

" in the custom container, added to the node name." I am... not sure what this means. We're talking about custom containers, but this is Control's description. The subject of "added to the node name" is ambiguous too.